kvs/src/benchmark/benchmark.cpp

```diff
@@ -75,10 +76,53 @@ int sample(int n, unsigned& seed, double base,
   return zipf_value;
 }
 
+unsigned get_total_address_size(
+    std::unordered_map<unsigned, std::unordered_set<Address>>& addresses) {
+  unsigned count = 0;
+  for (const auto& tier_address_pair : addresses) {
+    count += tier_address_pair.second.size();
+  }
+  return count;
+}
+
+// For GET, first check shared memory, then memory, then EBS
```

Comment:
Why is it different between the two?

Reply:
In general, I think shared memory tier is more efficient in processing reads (negligible synchronization + all threads in a shared memory node can serve reads) and private memory tier is more efficient in processing writes (no synchronization). So that's why there's a difference